### PR TITLE
✨Add requestCount, maxScrollDepth, and totalTime variables to Chartbeat vendor config

### PR DESCRIPTION
**Pull request by joshschwartz**
This PR adds three new variables to the Chartbeat vendor config to keep it in sync with our non-AMP scripts.

### CODE REVIEW

**Comment by joshschwartz** [comment]
Tagging @ampproject/wg-analytics for review

**Comment by torch2424** [comment]
@calebcordry Restarted what may have been a flaky test for you 😄

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   },
   "chartbeat": {
     "host": "https://ping.chartbeat.net",
-    "basePrefix": "/ping?h=!domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=!uid&g0=!sections&g1=!authors&g2=!zone&g3=!sponsorName&g4=!contentType&c=120&x=_scroll_top_&y=_scroll_height_&o=_scroll_width_&w=_viewport_height_&j=!decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&C=2",
+    "basePrefix": "/ping?h=!domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=!uid&g0=!sections&g1=!authors&g2=!zone&g3=!sponsorName&g4=!contentType&c=_total_time_&x=_scroll_top_&m=_max_scroll_depth_&y=_scroll_height_&o=_scroll_width_&w=_viewport_height_&j=!decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&sn=_request_count_&C=2",
```
Comment (calebcordry) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: `_total_time_ => !totalTime`
`_max_scroll_depth_ => !maxScrollDepth`
`_request_count_ => 1`

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
...
     "baseSuffix": "&_",
     "interval": "https://ping.chartbeat.net/ping?h=!domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=!uid&g0=!sections&g1=!authors&g2=!zone&g3=!sponsorName&g4=!contentType&c=120&x=_scroll_top_&y=_scroll_height_&o=_scroll_width_&w=_viewport_height_&j=!decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&C=2&_",
```
Comment (calebcordry) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: need to add these new vars to `interval` here and `anchorClick` below.

**Comment by mrjoro** [comment]
@joshschwartz, please make the test update @calebcordry described so we'll be able to get this in.  Thanks!

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 3]
```diff
@@ -54,11 +54,11 @@
...
-  },
+    "interval": "https://ping.chartbeat.net/ping?h=!domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=!uid&g0=!sections&g1=!authors&g2=!zone&g3=!sponsorName&g4=!contentType&c=!totalTime&x=_scroll_top_&m=!maxScrollDepth&y=_scroll_height_&o=_scroll_width_&w=_viewport_height_&j=!decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&sn=1&C=2&_",
+    "anchorClick": "https://ping.chartbeat.net/ping?h=!domain&p=_canonical_path_&u=_client_id_&d=_canonical_host_&g=!uid&g0=!sections&g1=!authors&g2=!zone&g3=!sponsorName&g4=!contentType&c=!totalTime&x=_scroll_top_&m=!maxScrollDepth&y=_scroll_height_&o=_scroll_width_&w=_viewport_height_&j=!decayTime&R=1&W=0&I=0&E=_total_engaged_time_&r=_document_referrer_&t=_page_view_id__client_id_&b=_page_load_time_&i=_title_&T=_timestamp_&tz=_timezone_&sn=1&C=2&_"
+  }
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: `,`

**Comment by tommcgurl** [comment]
@googlebot I fixed it.

**Comment by tommcgurl** [comment]
@googlebot I fixed it.

**Comment by tommcgurl** [comment]
Thanks for the review/help @calebcordry and @micajuine-ho 👍 . I've pushed the updates and it looks like everything is passing 🎉

**Comment by tommcgurl** [comment]
Tagging @ampproject/wg-analytics for review

**Comment by joshschwartz** [comment]
Sorry this has been open for so long! Should be all set, so tagging @ampproject/wg-analytics for review

**Pull request merged**